swagger2.0/oic.wk.con.p.swagger.json

```diff
@@ -148,12 +144,13 @@
           "type": "array"
         }
       },
-      "type" : "object"
+      "type" : "object",
+      "required": ["n"]
     },
-    "Update_Platform" : {
+    "Update_Platform": {
       "properties": {
-        "n" :  {
-          "description": "Friendly name of the resource",
+        "n":  {
```

Comment:
I think I get what this is doing now.  However, the way to change the value of "n" is with oic.wk.con, not oic.wk.con.p, right?  I guess this isn't explained very clearly in the spec but the reference to "n" above ("$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n") should lead to a schema where "n" is read only, not RW.

It feels like "n" as a RW Property should not be in this schema.